cmd/ctl/alpha/artifact.go

```diff
@@ -0,0 +1,64 @@
+/*
+Copyright 2022 The KubeSphere Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package alpha
+
+import (
+	"github.com/kubesphere/kubekey/cmd/ctl/options"
+	"github.com/kubesphere/kubekey/cmd/ctl/util"
+	"github.com/kubesphere/kubekey/pkg/alpha"
+	"github.com/kubesphere/kubekey/pkg/common"
+	"github.com/spf13/cobra"
+)
+
+type ArtifactImportOptions struct {
+	CommonOptions *options.CommonOptions
+	Artifact      string
+}
+
+func NewUpgradePrecheckOptions() *ArtifactImportOptions {
+	return &ArtifactImportOptions{
+		CommonOptions: options.NewCommonOptions(),
+	}
+}
+
+// NewCmdArtifactImport creates a new artifact import command
+func NewCmdArtifactImport() *cobra.Command {
```

Comment:
Is there a need to add a `Validate()` or `Complete()` method to check these `args` and `flags` provided by the user?
What happened if a user doesn't provide a path to an `artifact`?

For example:
https://github.com/kubesphere/kubekey/blob/97ad9f358074647a1b21d7d23403b85cf70b7949/cmd/ctl/artifact/images/push.go#L67
https://github.com/kubesphere/kubekey/blob/97ad9f358074647a1b21d7d23403b85cf70b7949/cmd/ctl/artifact/images/push.go#L79

Reply:
Oh, I forgot to handle the error. I will use Complete() to solve it.

Reply:
It happens often. 🤣